examples/todo-mvc/ui/views/task_list_view.slint

```diff
@@ -46,6 +46,7 @@ export component TaskListView {
         SelectionListView {
             width: 100%;
             model: TaskListAdapter.tasks;
+            accessible-label: @tr("task listview");
```

Comment:
@DataTriny Could you help us out here: Should accessible labels be capitalised? (my feeling is yes, but it would be good to confirm :)

Reply:
To be fair, capitalization is the least of my concerns here. Why do you talk about "listview" to your users? "List of tasks" would be far more appropriate.